backend/tracim_backend/exceptions.py

```diff
@@ -319,6 +319,10 @@ class EmptyLabelNotAllowed(EmptyValueNotAllowed):
     pass
 
 
+class UnallowedWorkspaceAccessType(TracimException):
```

Comment:
```suggestion
class DisallowedWorkspaceAccessType(TracimException):
```

Reply:
Renamed everywhere, Change the Error name too.